platform/macos/app/MapDocument.m

```diff
@@ -182,6 +182,12 @@ - (IBAction)showStyle:(id)sender {
         case 6:
             styleURL = [MGLStyle satelliteStreetsStyleURL];
             break;
+        case 7:
+            styleURL = [NSURL URLWithString:@"mapbox://styles/mapbox/traffic-day-v2"];
+            break;
+        case 8:
+            styleURL = [NSURL URLWithString:@"mapbox://styles/mapbox/traffic-night-v2"];
```

Comment:
To clarify, it’s fine to remove the deprecated styles from macosapp – after all, Emerald isn’t listed – but you need to remove them completely. Otherwise, any inconsistencies between this code and the XIBs could result in a crash.

Reply:
Removed or kept, we should do the same here and in iosapp.